action.yaml

```diff
@@ -1,22 +1,27 @@
 name: 'Iter8 Install'
 description: 'Install Iter8 CLI'
+inputs:
+  version:
+    description: "The iter8 version to use (default: the git reference of the installed action itself)"
+    required: true
+    default: "${{ github.action_ref }}"
 runs:
   using: 'composite'
   steps:
     - name: Install latest version of Iter8 CLI
       env:
-        VERSION: ${{ github.action_ref }}
-      run: | 
+        VERSION: ${{ inputs.version }}
```

Comment:
Is `env.Version` still required? Shouldn't we replace `env.Version` with `inputs.version`?

Reply:
Also, we have gone back-and-forth on the following idea. But, SHA pinning of Iter8 action seems like a good reason to have a separate repo for the iter8 action altogether. Then we can ... 

```yaml
-uses: iter8-tools/iter8-action@<sha commit>
```

And within the `action.yaml`, we can fix the Iter8 release that is getting used to install Iter8!

FYI @kalantar.